README.md

```diff
@@ -256,6 +255,46 @@ docker run --rm -it --net=host -v ${PWD}:/nConnect/data nknorg/nconnect
 
 followed by the command line argument you want to add.
 
+## nConnect Client Connects to Multi Servers
+
+Now nConnect client can connect to multi servers. You can edit `config.json` to add multi servers admin address:
+
+```
+
+{
+  "Client": true,
+  "Server": false,
+  "identifier": "alice",
+  "seed": "",
+  "remoteAdminAddr": [
+      "nConnect.bob1.7cafe0ae02789f8eb6b293e46b0ac5cf8f92f73042199c8161e5b5f90b13dcb5",
+      "nConnect.bob2.7cafe0ae02789f8eb6b293e46b0ac5cf8f92f73042199c8161e5b5f90b13dcb5",
+      "nConnect.bob3.7cafe0ae02789f8eb6b293e46b0ac5cf8f92f73042199c8161e5b5f90b13dcb5",
+  ],
+  "localSocksAddr": "127.0.0.1:1080",
+}
+
+```
+
+After config multi `remoteAdminAddr`, the nConnect client will add routing infomation to each Servers' local IP. So you can access all the servers by their local IP address.
+
+Specifically, the first  item in the `remoteAdminAddr` will become the default server which will get the forwarded data which targets are beyond all these servers' local IP address. Such as access to website by domain, or some other applications.
+
+You can use command argument to connects to multi servers too. argument `-a` witch address separated by comma "," to pass multi servers addresses:
+
+```
+
+$ nConnect -c -a "server-address1, server-address2, server-address3"
```

Comment:
Should it be `-a addr1 -a addr2 -a addr3`?

Reply:
Yes, modified.